gcp/deployment-packages/autoscale-payg/README.md

```diff
@@ -0,0 +1,130 @@
+# GCP Deployment Manager package for Check Point Autoscaling PAYG solution
```

Comment:
if the difference between byol and payg is only in the one imported python file, maybe put it in the same directory?